fluentui_transients/src/main/java/com/microsoft/fluentui/tooltip/Tooltip.kt

```diff
@@ -295,12 +302,12 @@ class Tooltip {
         val startPosition = positionX + layoutParams.marginStart
         val topBarHeight = context.statusBarHeight + ( context.activity?.supportActionBar?.height ?: 0 )
         val doesNotFitAboveOrBelow = (positionY < topBarHeight) || (positionY + contentHeight > displayHeight)
-        val rightSpace = displayWidth - anchorRect.right - context.softNavBarOffsetX
-        val rightEdge = ( startPosition + upArrowWidth + cornerRadius + margin > displayWidth ) || (doesNotFitAboveOrBelow && anchorRect.left > rightSpace)
+        val rightSpace = displayWidth - anchorRect.right + context.softNavBarOffsetX
+        val rightEdge = ( startPosition + upArrowWidth + cornerRadius + margin - context.softNavBarOffsetX > displayWidth ) || (doesNotFitAboveOrBelow && anchorRect.left > rightSpace)
         val leftEdge = ( startPosition - cornerRadius - margin - context.softNavBarOffsetX < 0 ) || (doesNotFitAboveOrBelow && anchorRect.left < rightSpace)
```

Comment:
I believe this should be removed then?

Reply:
softNavBarOffset is only calculated at 270degree rotation, 0 otherwise. So this should be good.
![image](https://user-images.githubusercontent.com/88475350/140008320-969d1de7-766e-44d1-8691-37d5119fbbd8.png)